core/src/main/scala/pureconfig/ConfigCursor.scala

```diff
@@ -276,24 +301,30 @@ object ConfigCursor {
 
   /** Builds a `ConfigCursor` for a `ConfigValue` at a path.
     *
-    * @param value the `ConfigValue` to which the cursor should point
-    * @param pathElems the path of `value` in the config as a list of keys
-    * @return a `ConfigCursor` for `value` at the path given by `pathElems`.
+    * @param value
+    *   the `ConfigValue` to which the cursor should point
+    * @param pathElems
+    *   the path of `value` in the config as a list of keys
+    * @return
+    *   a `ConfigCursor` for `value` at the path given by `pathElems`.
     */
   def apply(value: ConfigValue, pathElems: List[String]): ConfigCursor = SimpleConfigCursor(Option(value), pathElems)
 
   /** Builds a `ConfigCursor` for an optional `ConfigValue` at a path.
     *
-    * @param value the optional `ConfigValue` to which the cursor should point
-    * @param pathElems the path of `value` in the config as a list of keys
-    * @return a `ConfigCursor` for `value` at the path given by `pathElems`.
+    * @param value
+    *   the optional `ConfigValue` to which the cursor should point
+    * @param pathElems
+    *   the path of `value` in the config as a list of keys
+    * @return
+    *   a `ConfigCursor` for `value` at the path given by `pathElems`.
     */
   def apply(value: Option[ConfigValue], pathElems: List[String]): ConfigCursor = SimpleConfigCursor(value, pathElems)
 
   /** Handle automatic type conversions of `ConfigValue`s the way the
-    * [[https://github.com/lightbend/config/blob/master/HOCON.md#automatic-type-conversions HOCON specification]]
```

Comment:
@ruippeixotog is this a standard space before `HOCON`, or some unicode variation?